drivers/camera1394/launch/downward.launch

```diff
@@ -0,0 +1,50 @@
+<launch>
+<group ns="downward">
+    <node pkg="nodelet" type="nodelet" name="camera_nodelet_manager"
+          args="manager" />
+
+    <!-- Left/Right Camera Driver Nodelets -->
+
+    <!-- DOWNWARD CAMERA-->
+    <node pkg="nodelet" type="nodelet" name="down_camera1394_nodelet"
+          args="load camera1394/driver camera_nodelet_manager">
+          <param name="guid" value="00b09d01009ce76f"/>
+	  <param name="camera_info_url" value=""$(find camera1394)/calibration/${NAME}"/>
```

Comment:
Hey Ralph, this line seems to break the launch file (I think the double-quotes after value= are causing that)